index.html

```diff
@@ -12,25 +12,30 @@
 </head>
 <body>
     <div class="menu">
-        <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
-        <h1>Your forecast app</h1>
-        <div class="search">
-            <span class="enter">enter your city
-                <div class="search__icon"><img src="icons/search.png" alt=""></div>
-            </span>
+        <div class="block__menu">
+            <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
+            <h1 class="title__one">Your forecast app</h1>
+            <div>
+                <div>
+                    <form name="search" method="post">
+                        <input class="enter" type="text" value="enter your city">
+                        <div class="search__icon"><a href="#"><img src="icons/search.png" alt=""></a></div>
+                    </form>
+                </div>
+            </div>
+            <h3 class="title__three"><a href="#">give me a random one</a></h3>
         </div>
-        <h3>give me a random one</h3>
     </div>
     <div class="london">
+        <div class="back"></div>
+        <div class="back__search"></div>
```

Comment:
1. Use a `button` tag if the component involves pressing.
2. Wrap these buttons in a container.
3. Rename classes with BEM notation